fastlane/README.md

```diff
@@ -0,0 +1,29 @@
+fastlane documentation
+================
+# Installation
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111)